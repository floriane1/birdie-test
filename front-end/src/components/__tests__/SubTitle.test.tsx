import * as React from 'react';
import { shallow, configure } from 'enzyme';
import * as Adapter from 'enzyme-adapter-react-16';
import SubTitle from '@App/components/SubTitle';

configure({ adapter: new Adapter() });

describe('SubTitle', () => {
  it('renders correctly', () => {
    const wrapper = shallow(
      <SubTitle>Welcome to the Birdie platform</SubTitle>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
