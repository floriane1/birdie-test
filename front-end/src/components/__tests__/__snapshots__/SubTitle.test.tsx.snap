// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SubTitle renders correctly 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": false,
        "rules": Array [
          "
      font-family: sans-serif;
      font-size: 18px;
      margin-top: 0;
    ",
        ],
      },
      "displayName": "styled.h3",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "h3",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
>
  Welcome to the Birdie platform
</StyledComponent>
`;
